README.md

```diff
@@ -1,7 +1,49 @@
-I Am Open Source
+i am open source
 ===
 
-### TODOS:
-- [ ] Figure out contribute link
-- [ ] Figure out content for why page
-- [ ] Clean up Jekyll site a bit
+Getting started with open source can be difficult--[i am open source](https://iamopensource.github.io/) is a place to show new comers that anyone (from any background) can contribute just about anything from a single character typo to a whole new feature. We use the GitHub API to pull in first time contributions from repos that use [behaviorbot](https://github.com/behaviorbot/welcome) to welcome new contributors and we currate stories from individuals who have jumped into open source.  
+
+### Background
+
+[@hiimbex](https://github.com/hiimbex) wrote a blog post called [Gaining my voice in open source](https://medium.com/@hiimbexo/gaining-my-voice-in-open-source-21c8c1ac0eee) and it really resonated with a lot of first timers. Bex raised a couple of concerns that seemed to plague a lot of us who would like to contribute but haven't yet found solid footing to get started. Ultimately a fear of not having previous contributions and a difficulty identifying with the majority of the community left open source projects devoid of Bex's unique voice and exctiment to contribute. This is a huge missed opportunity for the community and for Bex. [@hiimbex](https://github.com/hiimbex) and [@anglinb](https://github.com/anglinb) set out to help fix this. 
+
+### Mission
+
+#### Dispelling Contribution Misconceptions
+
+A lot of new comers to open source have an idea of what "a contribution" is and what their excepted role is supposed to be. However, many of these definitions limit new comers--fixing a typo in docs or embellishing an under-documented feature is something that no one wants to do but will greatly help future contributors. These kinds of contributions are perfect for new comers because fixing docs is still exciting and command over a language like English might come more natural than command over a programming language.
```

Comment:
Maybe here add "Some people's first open source contribution is helping translate documentations to make a project more accessible to everyone."

Helps prevent us from seeming like english is the only "acceptable" way to do open source